develocity-maven-extension-adapters/build.gradle.kts

```diff
@@ -32,3 +32,9 @@ develocityAdaptersPublication {
     name = "Develocity API adapters for Develocity and Gradle Enterprise Maven extensions"
     description = "Adapter interfaces for interacting with the Develocity and Gradle Enterprise Maven extension APIs"
 }
+
+tasks.withType<Jar>().configureEach {
+    from(file("../LICENSE")) {
+        into("META-INF")
```

Comment:
❔ I saw Eric mentioning that the LICENSE file should be in the root of the JAR. But here it is not. Does it apply only to our Gradle deliverables?

Reply:
No, it applies to all, this was just missed...fixed it